go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/skip-mev/slinky
 
 go 1.21
 
+replace github.com/cosmos/cosmos-sdk => github.com/Eric-Warehime/cosmos-sdk v0.50.3-0.20240129022645-0cc581697f7e
```

Comment:
Why do we need this `replace`?

Reply:
👍 Leftover from some debugging I was doing.